skfp/fingerprints/avalon.py

```diff
@@ -12,7 +12,64 @@
 
 
 class AvalonFingerprint(FingerprintTransformer):
-    """Avalon fingerprint."""
+    """
+    Avalon fingerprint.
+
+    The implementation uses RDKit [1]_. This is a hashed fingerprint, where
+    fragments are computed based on atom environments. Avalon is a path-based
+    fingerprint.
```

Comment:
I would say that fragments are based on multiple feature classes, including e.g. atom and bond type counts, rings and paths. In general, Avalon has quite a large collection of different fragments, and even their combinations.

Reply:
Maybe add "The detailed descriptions of feature classes are in the supplementary material of [3]_", and this will replace the "The fingerprint is based..."